### PR TITLE
Feature branch - Building with Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by Yors-git**
- Created a clone of an old Apple home page
- We used Flex, Grid, Float for positioning and layout
- We used gradients and background images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                    <i class="far fa-play-circle fa-2x"></i>
+                </a>   
+            </div>
+        </div>
+        <div class="bottom-images">
+            <img src="./img/promo_iphone5s_2x.jpg" alt="iphone 5s image">
+            <img src="./img/promo_iphone5c_2x.jpg" alt="iphone 5c image">
+            <img src="./img/promo_30_years_2x.jpg" alt="apple 30 years image">
+            <img src="./img/promo_supplier_responsibility_2x.jpg" alt="supplier responsibility">
+        </div>
```
Comment (krys2fa) [inline index.html]: - [x] [REQUIRED] This section should contain four elements which are links (i.e either to other articles or `#`)

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome job @meronokbay and @Yors-git  👏 
Your project displays good use of semantic HTML as well as CSS classes. 💪 
You have a professional-looking README file as well. 👍 

Please make the following changes to complete the project. They can be seen below:
### Status: Changes Required ♻️ 

- [x] [REQUIRED] You are required to set up code linters in your project directory ([pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/)).
You only have the linter for GitHub actions at the moment.   


Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

File: index.html (hunk 1 above)
Comment (Yors-git) [reply index.html]: Fixed, we made images act as links

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: :+1: 
Great work making the changes as requested in the previous review.
Your project is now ready
On to the next one.

### HAPPY CODING :computer:

**Pull request merged**